index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="bookApp">
+<head>
+  <title>Angular Book App</title>
+
+  <!-- bootstrap script -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+  <!-- stylesheets -->
+  <link rel="stylesheet" href="/public/styles/styles.css">
+
+  <!-- angular scripts -->
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular-resource.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.2/angular-route.min.js"></script>
+
+  <!-- application scripts -->
+  <script src="/public/scripts/app.js"></script>
+  <script src="/public/scripts/controllers/BooksIndexController.js"></script>
+  <script src="/public/scripts/controllers/BooksShowController.js"></script>
+
+
+</head>
+<body>
+
+  <div class="jumbotron">
```

Comment:
why do you have both jumbotron and container? is that just copy-pasted from an old assignment?